libraries/AP_NavEKF2/AP_NavEKF2_Measurements.cpp

```diff
@@ -251,7 +251,7 @@ void NavEKF2_core::readIMUData()
     } else {
         readDeltaAngle(ins.get_primary_gyro(), imuDataNew.delAng);
     }
-    imuDataNew.delAngDT = MAX(ins.get_delta_time(),1.0e-4f);
+    imuDataNew.delAngDT = MAX(ins.get_delta_angle_dt(imu_index),1.0e-4f);
```

Comment:
I did it in the way in the PR to make it match get_delta_velocity(), which doesn't return the time. 
There is a separate get_delta_velocity_dt() function